src/backend/renderer/color.rs

```diff
@@ -0,0 +1,66 @@
+/// A four-component color
```

Comment:
Nitpick: Can we document that this is supposed to be pre-multiplied?
I don't think we handle colors in any other way anywhere in smithay and once we do I would use the type-system anyway to make misuse harder.

Reply:
Yeah, I started with some generic `Color` type, but opted out pretty quickly... :sweat_smile:

Reply:
done